src/types/account-id.ts

```diff
@@ -38,6 +42,10 @@ class AccountID extends Hash160 {
    * @returns an AccountID object
    */
   static fromBase58(value: string): AccountID {
+    if (!isValidClassicAddress(value)) {
```

Comment:
Is this necessary? I think `decodeAccountID` throws an error if the value is not a valid classic address.

Reply:
Okay, if `decodeAccountID` throws then we don't need it.

Reply:
Let me look into it real quick